intTests/test1894/test.saw

```diff
@@ -0,0 +1 @@
+(\n -> n) 1;
```

Comment:
This needs an accompanying `test.sh` file, or else the test suite won't run it.

Reply:
I think that should be there already?